### DISCUSSION TITLE
Support for Python 3.7 + 1.x

### DISCUSSION THREAD

**Discussion by gogasca** [jupyter/jupyter_server#1202] [General]
Thanks for the fix here: https://github.com/jupyter-server/jupyter_server/pull/1180
Unfortunately our team has a dependency on Python 3.7 hence we can't upgrade to version `jupyter_server` 2.x for a while. (Need to migrate/test fix other dependencies)

Is there a way we can include this fix for version 1.X which my understanding still supports 3.7? 

Reference: https://github.com/jupyter-server/jupyter_server/pull/1035#discussion_r1009694849

Thanks

**Comment by gogasca** [comment]
Is the right way to go about this is to submit a back-port PR under 1.x branch? https://github.com/jupyter-server/jupyter_server/tree/1.x

**Comment by blink1073** [comment]
Hi @gogasca, yes, following these instructions: https://github.com/jupyter-server/jupyter_server/pull/1180#issuecomment-1419498592

**Reply by gogasca** [reply]
Thanks for responding, submitted: https://github.com/jupyter-server/jupyter_server/pull/1204
